src/subcommands/logs.ts

```diff
@@ -24,85 +33,224 @@ OPTIONS:
         --prod                        Select the production deployment
     -p, --project=NAME                The project you want to get the logs
         --token=TOKEN                 The API token to use (defaults to DENO_DEPLOY_TOKEN env var)
+        --since=<DATETIME>            The start time of the logs you want to get. RFC3339 format (e.g. 2023-07-17T06:10:38+09:00) is supported.
+                                      NOTE: Logs generated over 24 hours ago are not available
+        --until=<DATETIME>            The end time of the logs you want to get. RFC3339 format (e.g. 2023-07-17T06:10:38+09:00) is supported.
+        --grep=<WORD>                 Filter logs by a word
```

Comment:
We should make it clear that multiple words will result in AND operation on the log. I.e. A log should have `wordA` AND `wordB` when a user specifies `--grep=wordA --grep=wordB` for a match.

Reply:
Good point, I added that in [c47b47a](https://github.com/denoland/deployctl/pull/165/commits/c47b47a8ddd5dd1b224946ba0e00e079814f91b8)